content/articles/creating-serverless-functions-faas-with-aws-lambda-and-golang/index.md

```diff
@@ -0,0 +1,292 @@
+Creating Serverless Functions (Faas) with AWS Lambda and Golang
+
+Serverless architecture is a new paradigm that is being used in the development of modern applications. The term "serverless" refers to the abstraction of "servers" from application development. Function-as-a-Service (FaaS) is a serverless method of running modular code on the cloud. You can use FaaS to write code that is triggered by events. For example, when a user clicks on a button in your web application, a specific cloud function is executed.
+
+We will not only look at how to use FaaS in this tutorial, but we will also construct some for others to use. We'll be working in Golang, but the same method can be applied to Python, Node, Java, or any other language or framework you want.
+
+### Requirements
+
+* [AWS](https://aws.amazon.com) account
+* Go 1.x installed on your machine
+
+### Setting up a Lambda Function on AWS
+
+<<<<<<< HEAD
+1. After Creating your AWS account, on the main dashboard, search for Lambda.![search-aws-lambda](search-aws-dashboard.png)When you click on Lamda, the AWS lambda will be visible.![aws-lambda-dashboard](aws-lambda-dashboard.png)
+2. Click on the Create Function button to set up a new AWS Lambda function.![aws-create-lambda-setting](aws-create-lambda-setting.png)The name of the function is user-profile. Click on Create function.
+3. On the function’s dashboard, we will upload the function we want to run in the cloud. ![aws-lambda-function-dashboard](aws-lambda-function-dashboard.png)In the next section, we'll look at how to make a function that runs on AWS Lambda.
+=======
+1. After Creating your AWS account, on the main dashboard, search for Lambda.![search-aws-lambda](/engineering-education/creating-serverless-functions-faas-with-aws-lambda-and-golang/search-aws-dashboard.png)When you click on Lamda, you will see the AWS lambda page.![aws-lambda-dashboard](/engineering-education/creating-serverless-functions-faas-with-aws-lambda-and-golang/aws-lambda-dashboard.png)
+2. Click on the Create Function button to set up a new AWS Lambda function.![aws-create-lambda-setting](/engineering-education/creating-serverless-functions-faas-with-aws-lambda-and-golang/aws-create-lambda-setting.png)The name of the function is user-profile. Click on Create function.
+3. On the function’s dashboard, we will upload the function we want to run in the cloud. ![aws-lambda-function-dashboard](/engineering-education/creating-serverless-functions-faas-with-aws-lambda-and-golang/aws-lambda-function-dashboard.png)In the next section, we will see how to create a function that will run on AWS lambda.
+>>>>>>> 012ab3bce84482631c2308ae86a7bca9c5bbe029
+
+### Creating the User-Profile Function in Golang
+
```

Comment:
Add a section that talks about the objective of the tutorial. What is the expected end result? What are you going to build?

Reply:
"We will not only look at how to use FaaS in this tutorial, but we will also construct some for others to use through AWS. We'll be working in Golang, but the same method can be applied to Python, Node, Java, or any other language or framework you want."

Looks like I have done this already.

Reply:
@Bamimore-Tomi 

Not sure if I was clear.

What I really meant is - you were building a user profile. I want you to state the objective of the code (not the entire tutorial), the expected end result of the code.

`construct some for others to use through AWS`

What exactly do you mean by this?

Reply:
Alright, thank you.

I have made changes.